regression/scripts/weights/eVTOL_Weights_Buildup_Regression.py

```diff
@@ -0,0 +1,352 @@
+# eVTOL_Weights_Buildup_Regression.py
+#
+# Author: Jordan Smart, Feb. 2018
+
+import sys
+import SUAVE
+import numpy as np
+import pprint as pp
+from SUAVE.Methods.Weights.Buildups.electricHelicopter import empty as electricHelicopterEmpty
+from SUAVE.Methods.Weights.Buildups.electricStoppedRotor import empty as electricStoppedRotorEmpty
+from SUAVE.Methods.Weights.Buildups.electricTiltrotor import empty as electricTiltrotorEmpty
+from SUAVE.Components.Energy.Networks.Battery_Propeller import Battery_Propeller
+from SUAVE.Methods.Propulsion import propeller_design
+from SUAVE.Methods.Power.Battery.Sizing import initialize_from_mass
+from SUAVE.Core import (
+    Data, Container, Units,
+)
+
+
+#-------------------------------------------------------------------------------
+# Define Base Vehicle
+#-------------------------------------------------------------------------------
+
+def vehicle_setup():
+    vehicle = SUAVE.Vehicle()
+    vehicle.tag = 'eVTOL Weight Test'
+    
+    #---------------------------------------------------------------------------
+    # Vehicle-Level Properties
+    #---------------------------------------------------------------------------
+    
+    vehicle.mass_properties.takeoff         = 840.  * Units.kg
+    vehicle.mass_properties.operating_empty = 640.  * Units.kg
+    vehicle.mass_properties.max_takeoff     = 1000. * Units.kg
+    
+    vehicle.reference_area                  = 12.
+    
+    #---------------------------------------------------------------------------
+    # Fuselage Properties Properties
+    #---------------------------------------------------------------------------
+    
+    fuselage = SUAVE.Components.Fuselages.Fuselage()
+    fuselage.tag = 'fuselage'
+    
+    fuselage.lengths.total      = 5.5
```

Comment:
Units? Not necessary, but might look more consistent.

Reply:
Added.